src/components/WhaleInfo/WhaleInfo.scss

```diff
@@ -1,17 +1,13 @@
 @import "../../Constants.scss";
 
 // Temporary border constants for designing. Remove or set to transparent 0px after use
-$border: 2px solid rgb(255, 0, 0);
+$border: 2px solid $darker-background;
```

Comment:
Do we still want these temp borders on this page?

Reply:
We want to leave these borders as part of formatting so will delete the comment suggesting they are temporary now. 

Will discuss this consistency in the design meeting.